examples/tutorials/08_research_tutorial/2_Research_advanced.ipynb

```diff
@@ -0,0 +1,1030 @@
+{
```

Comment:
<p>Do not remove files silently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>